docs/pre-release/install/upgrade-tabs.js

```diff
@@ -97,6 +99,19 @@ export default function SimpleTabs() {
         }
         </CodeBlock>
       </TabPanel>
+      <TabPanel value={value} index={2}>
+        <CodeBlock>
+        {
+          '# 1. Download the latest binary (~50 MB):' +
+          '\n' +
+                'sudo curl -fL https://app.getambassador.io/download/tel2/windows/amd64/latest/telepresence.msi -o C:\\Temp\\telepresence.msi' +
```

Comment:
oh actually it looks like there's no `sudo` in windows; but i would think it's okay to run this without sudo; if `C:\Temp` isn't writeable, we can always use a different location